gravitee-apim-console-webui/src/components/gio-metadata/gio-metadata.component.ts

```diff
@@ -89,6 +89,7 @@ export class GioMetadataComponent implements OnInit, OnDestroy {
       .subscribe((metadata) => {
         this.dataSource = new MatTableDataSource(metadata);
         this.dataSource.sort = this.sort;
+        metadata['defaultValue'] && this.displayedColumns.push('defaultValue');
```

Comment:
Hard to understand, is it an if statement?

Reply:
Yes, so if `metadata['defaultValue'] ` exists do `this.displayedColumns.push('defaultValue'); `
But I can write it more clear with if statement

Reply:
Fixed